clients/android/NewsBlur/assets/dark_reading.css

```diff
@@ -1,22 +1,24 @@
-body, span {
+body, span, table, td, p, div {
 	background-color: #1A1A1A !important;
 	color: #FFF !important;
 }
 
 a, a span {
 	color: #319DC5 !important;
+	background-color: #1A1A1A !important;
 }
 
 a:visited, a:visited span {
 	color: #319DC5 !important;
+	background-color: #1A1A1A !important;
 }
 
-code {
+pre, blockquote, code {
     background-color: #4C4C4C;
 }
 
-pre, blockquote {
-    background-color: #4C4C4C;
+blockquote > p, pre > span, code span {
```

Comment:
Can you give me examples of the newsletters fixed by these selectors? I need to also implement this on iOS and possibly web.